docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -593,7 +593,7 @@ BESU_MIN_GAS_PRICE=1337
 min-gas-price=1337
 ```
 
-The minimum price that a transaction offers for it to be included in a mined block.
+The minimum price that a transaction offers for it to be included in a mined block. It is the same as the value returned by [`eth_gasPrice`](../API-Methods.md).
```

Comment:
Suggest updating this to reflect why you'd use `eth_gasPrice`. eg, 

Use `eth_gasPrice` to obtain the minimum gas price in a running node.  

ie, you use the `min-gas-price` CLI option to set the minimum gas price on startup but you can use `eth_gasPrice` to find out what it is in a running node.

Reply:
I added some of this language to the min-gas-price CLI option description and the eth_gasPrice method description.